REVIEWING.md

```diff
@@ -0,0 +1,32 @@
+# Reviewing and Merging Pull Requests
+
+This document is a guideline for how core contributors should review and merge
+pull requests (PRs). It is intended to outline the lightweight process that
+we’ll use for now. It’s assumed that we’ll operate on good faith for now in
+situations for which process is not specified.
+
+PRs may only be merged after the following criteria are met:
+
+1. It has been open for at least 1 business day
+1. It has no `veto` label on it
+1. It has been 'LGTM'-ed by 3 different reviewers, each from a different
```

Comment:
In either case, I believe @pmorie specifically used the phrase "gentle person's agreement--" which I take to mean that we're not formally documenting that bit of leniency.